helm/idol-nifi/values.schema.json

```diff
@@ -185,6 +191,54 @@
             ],
             "title": "Nifi"
         },
+        "NifiCluster": {
+            "type": "array",
+            "minItems": 1,
+            "items": {
+                "type": "object",
+                "additionalProperties": false,
+                "properties": {
```

Comment:
You can use `allOf` to reference the existing `#/definitions/NiFi` to reduce copying here.